packages/web-components/src/components/filter-panel/filter-panel-modal.ts

```diff
@@ -61,6 +121,80 @@ class DDSFilterPanelModal extends HostListenerMixin(
     );
   }
 
+  get focusableElements() {
+    const { selectorCloseButton, selectorTabbable: selectorTabbableForModal } =
+      this.constructor as typeof DDSExpressiveModal;
+    return [
+      ...Array.from(
+        (this.shadowRoot?.querySelectorAll(
+          selectorCloseButton
+        ) as NodeListOf<HTMLElement>) || []
+      ),
+      ...Array.from(
+        this.querySelectorAll(
+          selectorTabbableForModal
+        ) as NodeListOf<HTMLElement>
+      ),
+    ];
+  }
+
+  private get _focusablePanelModalElements() {
+    const { hasFocusableElements } = this;
+
+    const focusableElements: [HTMLElement?] = [];
+
+    hasFocusableElements.forEach((el) => {
+      if (el.focusableElements) {
+        focusableElements.push(...el.focusableElements);
+      }
+    });
+
+    return Array.from(new Set(focusableElements)).sort((a, b) => {
+      const comparison = a!.compareDocumentPosition(b!);
+
+      /* eslint-disable no-bitwise */
+      if (comparison & PRECEDING) {
+        return 1;
+      }
+
+      if (comparison & FOLLOWING) {
+        return -1;
+      }
+      /* eslint-enable no-bitwise */
+
+      return 0;
+    });
+  }
+
+  private _handlePanelModalFocusIn = ({ target, relatedTarget }) => {
```

Comment:
Same comment here, and theme throughout. Where we are copy / pasting code from the parent class b/c it's marked `private` there, could we instead open up the visibility to be protected or public so it could be inherited as is without duplication?